charts/etcd/templates/etcd-poddisruptionbudget.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: policy/v1
```

```suggestion
apiVersion: policy/v1beta1
```
PDBs were only graduated in Kubernetes 1.21 ([ref](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125)).